tortoise/backends/base/executor.py

```diff
@@ -72,27 +83,32 @@ def _prepare_insert_statement(self, columns: List[str]) -> str:
         raise NotImplementedError()  # pragma: nocoverage
 
     async def execute_insert(self, instance):
-        key = "{}:{}".format(self.db.connection_name, self.model._meta.table)
-        if key not in INSERT_CACHE:
-            regular_columns, columns = self._prepare_insert_columns()
-            query = self._prepare_insert_statement(columns)
-            INSERT_CACHE[key] = regular_columns, columns, query
-        else:
-            regular_columns, columns, query = INSERT_CACHE[key]
-
-        values = self._prepare_insert_values(instance=instance, regular_columns=regular_columns)
-        insert_result = await self.db.execute_insert(query, values)
+        values = [
+            self.column_map[column](getattr(instance, column), instance)
+            for column in self.regular_columns
+        ]
+        insert_result = await self.db.execute_insert(self.query, values)
         await self._process_insert_result(instance, insert_result)
         return instance
 
+    async def execute_bulk_insert(self, instances):
+        valueses = [
+            [
+                self.column_map[column](getattr(instance, column), instance)
+                for column in self.regular_columns
+            ]
+            for instance in instances
+        ]
+        await self.db.execute_many(self.query, valueses)
```

Comment:
I don't really think `valueses` is actual word  😄
Ma be `values_lists` would be better?

Reply:
I asked my wife, and she said a correct usage would be `values's` but `valueses` is just irritatingly wrong. Another option could be `values_of_values`, or shortened: `vovs` :rofl: 
I'll change it to `values_lists` :smile: